File: customfit-swift-client-sdk/Sources/CustomFitSwiftSDK/Core/Model/SdkSettings.swift
import Foundation

/// Simplified SdkSettings model with only essential flags
/// Only includes fields that are needed for core SDK functionality
public struct SdkSettings: Codable {
    public let cf_account_enabled: Bool
    public let cf_skip_sdk: Bool
    
    public init(cf_account_enabled: Bool = true, cf_skip_sdk: Bool = false) {
        self.cf_account_enabled = cf_account_enabled
        self.cf_skip_sdk = cf_skip_sdk
    }
} 

File: customfit-kotlin-client-sdk/src/main/kotlin/customfit/ai/kotlinclient/core/model/SdkSettings.kt
package customfit.ai.kotlinclient.core.model

import kotlinx.serialization.Serializable

/**
 * Simplified SdkSettings model with only essential flags
 * Only includes fields that are needed for core SDK functionality
 */
@Serializable
data class SdkSettings(
    val cf_account_enabled: Boolean = true,
    val cf_skip_sdk: Boolean = false
)
